team-01/documentation/linter-setup-instrutions.md

```diff
@@ -29,15 +29,15 @@ Setup:
 
 0. Pull the latest from main to have the styles.xml file locally. 
 1. Open intellij
-2. Open the preferences
-3. Open the code style tab
-4. Click on Java
-5. At the top of the view, you will see a "scheme" label. Click on the gear wheel on the right hand side of this scheme label
-6. Choose import scheme -> Eclipse XML profile
+2. Click `file` then `settings`
+3. Open the `code style` drop down menu
+4. Click on `Java`
+5. At the top of the view, you will see a `scheme` label. Click on the gear wheel on the right hand side of this scheme label
+6. Choose `import scheme` -> `Eclipse XML profile`
 7. Choose the file at location 322GameOfAmazons/style.xml
-8. Click OK
-9. In the "Tabs and Indents" tab, check the "Use tab character" option
-10. Set tab size to 4 instead of 2
-11. In the "Wrapping and Braces" tab, search for "Method annotations" and change that setting to "wrap always"
+8. Click `OK`
+9. In the `Tabs and Indents` tab, check the `Use tab character` option
+10. Set tab size from 2 to 4
+11. In the `Wrapping and Braces` tab, search for `Method annotations` and change that setting to `wrap always`
```

Comment:
I'd like to keep a set of instructions for me to be able to follow if I ever need to setup my environment again for some reason. Do you mind adding a "Windows" section with the Windows instruction instead of overwriting the current instructions?

Reply:
Not at all, great idea!

Reply:
Cool. To do it fast, if I were you, I'd probably copy paste the current content of the file, then revert your commit, then add the new section.

Reply:
But you do whatever works, it's just a suggestion. :)